### PR TITLE
Avoid "whitelist" according to rust-lang/rust changes

### PR DESCRIPTION
**Pull request by giraffate**
Fixed according to https://github.com/rust-lang/rust/commit/62cf767a4a39b47677d18110359d9e7152dc9d1c.

### CODE REVIEW

File: src/crates-io.md [hunk 1]
```diff
@@ -14,10 +14,10 @@ reasons:
 
 TODO: what is the vetting process?
 
-## Whitelist
+## Pemitted dependencies
 
-The `tidy` tool has a [whitelist] of crates that are allowed. To add a
-dependency that is not already in the compiler, you will need to add it to this
-whitelist.
+The `tidy` tool has [crates that are allowed]. To add a
```
Comment (mark-i-m) [inline src/crates-io.md:19]: perhaps "a list of crates..."?

Comment (giraffate) [reply src/crates-io.md]: You're right, thanks!

File: src/crates-io.md [hunk 2]
```diff
@@ -14,10 +14,10 @@ reasons:
 
 TODO: what is the vetting process?
 
-## Whitelist
+## Pemitted dependencies
```
Comment (jyn514) [inline src/crates-io.md:17]: Pemitted -> Permitted

Comment (giraffate) [reply src/crates-io.md]: Thanks!

File: src/crates-io.md [hunk 3]
```diff
@@ -14,10 +14,10 @@ reasons:
...
 
-## Whitelist
+## Permitted dependencies
 
-The `tidy` tool has a [whitelist] of crates that are allowed. To add a
-dependency that is not already in the compiler, you will need to add it to this
-whitelist.
+The `tidy` tool has [a list of crates that are allowed]. To add a
+dependency that is not already in the compiler, you will need to add it to these
+crates.
```
Comment (jyn514) [inline src/crates-io.md:21]:
```suggestion
dependency that is not already in the compiler, you will need to add it to the list.
```

File: src/stability.md [hunk 4]
```diff
@@ -76,11 +76,11 @@ Furthermore this attribute is needed to mark an intrinsic as callable from
 Macros, compiler desugarings and `const fn`s expose their bodies to the call
 site. To work around not being able to use unstable things in the standard
 library's macros, there's the `#[allow_internal_unstable(feature1, feature2)]`
-attribute that whitelists the given features for usage in stable macros or
+attribute that allows the given features for usage in stable macros or
```
Comment (jyn514) [inline src/stability.md:79]:
```suggestion
attribute that allows the given features to be used in stable macros or
```

**Pull request merged**